main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file_in:
+        players_data = json.load(file_in)
+
+    for player, data in players_data.items():
+
+        if not Race.objects.filter(name=data["race"]["name"]).exists():
+            Race.objects.create(
+                name=data["race"]["name"],
+                description=data["race"]["description"]
+            )
+
+        for skill_info in data["race"]["skills"]:
+            if not Skill.objects.filter(name=skill_info["name"]).exists():
+                Skill.objects.create(
```

Comment:
same as above

Reply:
![image](https://user-images.githubusercontent.com/88705381/216406740-71405c9b-ca96-4e98-827e-2ccbc4449fb1.png)

in this case, it is used only once. Does it make sense to save it into a variable? And if so how to do it coorectly in this case?

Reply:
Okey, you can remove `skill_name`.
Save object to variable only after creation, for example 
![image](https://user-images.githubusercontent.com/58142395/218116586-41c37317-1eb1-4605-8bb3-ddca7a76374a.png)
And then use it variable on 27, 42 and 43 lines